cvmfs/cloud_testing/run_remote_mac.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+usage() {
+  local msg="$1"
+
+  echo "Error: $msg"
+  echo "Usage: $0 <testee_url> <package_name> <source_name> <osx_name> <setup_script> <run_script>"
```

Comment:
you are relying on environment variables here, don't you? Please adapt the usage string accordingly.